1.wave2_data/3.processes_features/extract_image_features_interstellar.py

```diff
@@ -83,10 +87,60 @@
 image_features_df.head()
 
 
-# ## View info of the dataframe
+# ## Merge platemap metadata with extracted image features
```

Comment:
Does anything else happen with the code below besides merging (noticing that we used `annotate` and not a `pd.DataFrame.merge()`)? If so, consider providing this detail within the comment here.

Reply:
I don't think anything happens, I just wanted to stay consistent with the functions that are used with the other waves.

Is it better to just use this part of the function @d33bs 

```python
    # Load Data
    profiles = load_profiles(profiles)
    platemap = load_platemap(platemap, add_metadata_id_to_platemap)

    annotated = platemap.merge(
        profiles, left_on=join_on[0], right_on=join_on[1], how="inner"
    )
    if join_on[0] != join_on[1]:
        annotated = annotated.drop(join_on[0], axis="columns")
```